v5_MIGRATION_GUIDE.md

```diff
@@ -465,6 +461,15 @@ for (const client of clients.data) {
 // }
 ```
 
+Note that the multiple overridden functions for most `getAll` endpoints in v4 have been streamlined into one offset-based pagination function for each endpoint. For instance, v4's `clients.getAll()` has been replaced with `clients.list()` in v5, which is never overridden and only ever has the following signature:
```

Comment:
> into one offset-based pagination function for each endp

This is not correct. We use checkpoint by default where possible, and offset only for endpoints that have no checkpoint support.